_guidelines-v4/01-introduction/02-basicConcepts/04-meicustomization.md

```diff
@@ -0,0 +1,13 @@
+---
+version: "v4"
+title: "Customizing MEI"
+sectionid: "meicustomization"
+---
+
+In production, it is best to use a customized version of MEI, restricted to the very needs of a project. Such a custom schema will guide the encoders and will help to ensure consistency and data quality throughout a project's files. A customization typically provides a subset of MEI's encoding models (typically starting from one of the official _profiles_ mentioned in chapter {% include link id="meiprofiles" %}), with only one solution for any given situation being allowed. The customization will help to reflect the scope of a project into its data: Only those aspects of music notation a project is interested in will be allowed, so that the absence of a specific information can not be misunderstood as an oversight of the encoders. Larger editorial projects like _Complete Works_ editions typically use _Editorial Guidelines_ (german: _Editionsrichtlinien_) for the same purposes: (internal) quality control and (external) documentation. In that sense, MEI customizations may serve as Editorial Guidelines in digital form.
+
+MEI is implemented in ODD. ODD, or _One Document Does-it-all_, is another XML-based markup language developed and maintained by the TEI. TEI's documentation for ODD can be found in the TEI Guidelines chapter 22: [Documentation Elements](http://www.tei-c.org/release/doc/tei-p5-doc/en/html/TD.html){:target='\_blank'}, chapter 23: [Using the TEI](http://www.tei-c.org/release/doc/tei-p5-doc/en/html/USE.html){:target='\_blank'}, and the "[Getting Started with P5 ODDs](https://tei-c.org/guidelines/customization/getting-started-with-p5-odds/){:target='\_blank'}" document.
```

Comment:
documents

Reply:
not sure what you mean here – the linked "Getting Started with P5 ODDs" is one document, isn't it? Could you clarify?